### PR TITLE
:mortar_board: Topic combinational logic --- Mux

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Describe multiplexers and mention demultiplexers

### Testing

:+1: 

### Additional Notes

1. Implementing a demux could be a good lab activity 
2. The section feels like it should be a little longer, but I really do not know what else to say since I think the decoder gives enough of the insights in how these work.

### CODE REVIEW

File: site/topics/logic/combinational-logic.rst [hunk 1]
```diff
@@ -233,8 +233,268 @@ Decoder Symbol
...
+* Multiplexers (mux) are used to select one of many input signal to to map to an output
+
+    * The input signals could be arbitrary (high or low)
+    * They only have a single output
+
+
+* In addition to input signals, multiplexers have selector signals to control which input is mapped to the output
+* The below truth table demonstrates the desired functionality for a 2 bit mux
+
+.. list-table:: Two Bit Mux
```
Comment (jameshughes89) [inline site/topics/logic/combinational-logic.rst:248]: image

File: site/topics/logic/combinational-logic.rst [hunk 2]
```diff
@@ -233,8 +233,268 @@ Decoder Symbol
...
+      -
+      - ``1``
+      -
+      - ``1``
+
+
+* Truth tables like this may be simplified such that the values of the input signals are treated as some variable
+
+
+.. list-table:: Two Bit Mux Simplified
```
Comment (jameshughes89) [inline site/topics/logic/combinational-logic.rst:312]: image

File: site/topics/logic/combinational-logic.rst [hunk 3]
```diff
@@ -233,8 +233,268 @@ Decoder Symbol
...
+.. figure:: mux_demux_symbols.png
+    :width: 500 px
+    :align: center
+
+    Multiplexer with an output directly connected to a demultiplexer.
+
+
+* Below is a simplified truth table with the functionality of a demultiplexer
+
+.. list-table:: Four Bit Demux Simplified
```
Comment (jameshughes89) [inline site/topics/logic/combinational-logic.rst:441]: image

**Comment by jameshughes89** [comment]
I added screenshots of the rendered tables as it's going to be easier to read.

**Review by twentylemon — APPROVED** [review]

**Pull request merged**